lib/extensions/array_extension.rb

```diff
@@ -10,4 +10,8 @@ def zeroize_nils!(value = 0.00)
   def average(total = size)
     any? ? sum.to_f / total : 0
   end
+
+  def flat_select &block
```

Comment:
Rubocop is not happy with the method declaration :P 

Having the brackets surrounding the arguments should fix it:

```ruby
def flat_select(&block)
...
end
```